Feature/hooks show

ramiloif opened the pull request:
Show failed hooks ,
Works with new data added to report in PR https://github.com/adamgruber/mochawesome/pull/155

adamgruber commented on src/components/test/test.jsx:
```diff
@@ -37,7 +37,6 @@ class Test extends React.Component {
   render() {
     const { test, enableCode } = this.props;
     const { uuid, title, speed, duration, pass, fail, pending, skipped, err, code, context } = test;
-
```
Nit: leave space

adamgruber commented:
@ramiloif I made some changes that will affect this PR. You'll need to update the type definitions inside `bin/src/types.js` to include the new properties on `test` and `suite`. Also, when updating the tests, files inside `sample-data` will need to be updated to reflect the new data structure.

ramiloif commented:
Hi,
- Used isHook in order to show hooks differently 
- Added before and after hooks separation 
   In order to show then before or after the tests , this is associated with the PR on mochawesome 
   https://github.com/adamgruber/mochawesome/pull/157
- Removed the binary files .
- Added tests and changed the sample-data files
Here is how it looks like with the hooks 
<img width="1135" alt="screen shot 2017-05-06 at 2 49 51" src="https://cloud.githubusercontent.com/assets/15690633/25767945/c7047dea-3206-11e7-8c32-04a21e47d404.png">
<img width="1146" alt="screen shot 2017-05-06 at 2 49 36" src="https://cloud.githubusercontent.com/assets/15690633/25767946/c70d1cca-3206-11e7-9076-3818cd34a427.png">

adamgruber commented:
@ramiloif Thanks for your work on this. I used your PR as a starting point to add the feature. It's now available in v2.2.1
